package.json

```diff
@@ -7,7 +7,7 @@
     "@azure-tools/typespec-autorest": "0.36.1",
     "@azure-tools/typespec-azure-core": "0.36.0",
     "@azure-tools/typespec-azure-resource-manager": "0.36.1",
-    "@azure-tools/typespec-client-generator-core": "0.36.0",
+    "@azure-tools/typespec-client-generator-core": "0.36.1",
     "@azure-tools/typespec-csharp": "file:src/TypeSpec.Extension/Emitter.Csharp",
```

Comment:
Can we try to remove these duplicate version? I don't know why we have this.

Reply:
@pshao25 I don't know the purpose of this. Michael add this, maybe for float dependency. I suggest not to update this in this PR.